ts/components/screens/BaseHeader.tsx

```diff
@@ -36,7 +36,7 @@ export class BaseHeader extends React.PureComponent<Props> {
           </Left>
         )}
         <Body>
-          <Text white={this.props.primary} numberOfLines={1}>
+          <Text white={this.props.primary} numberOfLines={1} onPress={goBack}>
```

Comment:
@francescozaia I wonder if it would make more sense to move the label under the `<Left />` tag, what do you think?

Reply:
I've not enough knowledge of the app to answer, but on paper it makes sense. Will check that out.

Reply:
This cleanup requires quite a big rework. Let's push this task out of current scope and work on it separately.